### PR TITLE
#6 Task. Julia.D

### PR DESCRIPTION
**Pull request by JuliaDer**
https://travis-ci.org/github/JuliaDer/js-assignments/builds/707373332

### CODE REVIEW

File: task/06-conditions-n-loops-tasks.js [hunk 1]
```diff
@@ -30,7 +30,13 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 3 === 0 && num % 5 === 0) {
+        return 'FizzBuzz';
+    } else if (num % 5 === 0) {
```
Comment (KostyaKov) [inline task/06-conditions-n-loops-tasks.js]: "else" is redundant after "return"

File: task/06-conditions-n-loops-tasks.js [hunk 2]
```diff
@@ -46,7 +52,11 @@ function getFizzBuzz(num) {
  *   10 => 3628800
  */
 function getFactorial(n) {
-    throw new Error('Not implemented');
+    var ans = 1;
```
Comment (KostyaKov) [inline task/06-conditions-n-loops-tasks.js]: Need to use "let" for variables

File: task/06-conditions-n-loops-tasks.js [hunk 3]
```diff
@@ -296,7 +332,26 @@ function getDigitalRoot(num) {
...
+        '[':']',
+        '<':'>'
+    };
+    
+    for (let i = 0; i < str.length; i++) {
+      if (str[i] in open) { 
+        storage.push(str[i]);
+      }else { 
+        let recentOpen = storage.pop();
+        if(open[recentOpen] !== str[i]) {
```
Comment (KostyaKov) [inline task/06-conditions-n-loops-tasks.js]: It is necessary to observe indents from operators.

File: task/06-conditions-n-loops-tasks.js [hunk 4]
```diff
@@ -332,7 +387,23 @@ function isBracketsBalanced(str) {
...
+        time<=90?'a minute ago':
+            time<=45*min?`${Math.round(time/min)} minutes ago`:
+                time<=90*min?'an hour ago':
+                    time<=22*hour?`${Math.round(time/hour)} hours ago`:
+                        time<=36*hour?'a day ago':
+                            time<=25*day?`${Math.round(time/day)} days ago`:
+                                time<=45*day?'a month ago':
+                                    time<=345*day?`${Math.round(time/month)} months ago`:
+                                        time<=545*day?'a year ago':
+                                            `${Math.round(time/year)} years ago`;
```
Comment (KostyaKov) [inline task/06-conditions-n-loops-tasks.js]: For better readability of the code, it is preferable to use the "if", "else" statements

**Review by KostyaKov — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**